src/main/java/org/sonar/plugins/stash/CoverageCompat.java

```diff
@@ -0,0 +1,21 @@
+package org.sonar.plugins.stash;
+
+import org.sonar.api.issue.Issue;
+
+public final class CoverageCompat {
+  private final static String REPO = "coverageEvolution";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)